### PR TITLE
Version 4.1 to support Push to User

### PR DESCRIPTION
**Pull request by mpodwysocki**
Things to consider before you submit the PR:

* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [x] Did you add unit tests?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

This adds support for `UserId` and the new 2020-06 API

## Misc

Add what's missing, notes on what you tested, additional thoughts or questions.

### CODE REVIEW

File: src/Microsoft.Azure.NotificationHubs/Microsoft.Azure.NotificationHubs.csproj [hunk 1]
```diff
@@ -5,8 +5,9 @@
     <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <VersionPrefix>4.0.0.0</VersionPrefix>
+    <VersionPrefix>4.1.0.0</VersionPrefix>
     <PackageId>Microsoft.Azure.NotificationHubs</PackageId>
+    <PackageVersion>4.1.0-preview1</PackageVersion>
```
Comment (brannon) [inline src/Microsoft.Azure.NotificationHubs/Microsoft.Azure.NotificationHubs.csproj:10]: Is this intended to be a preview?

**Review by brannon — COMMENTED** [review]
Did we already make the API version change related to adding `userId` to the Installation body?

File: src/Microsoft.Azure.NotificationHubs/Microsoft.Azure.NotificationHubs.csproj (hunk 1 above)
Comment (mpodwysocki) [reply src/Microsoft.Azure.NotificationHubs/Microsoft.Azure.NotificationHubs.csproj:10]: No, this should be non-preview, will update

Comment (mpodwysocki) [reply src/Microsoft.Azure.NotificationHubs/Microsoft.Azure.NotificationHubs.csproj]: Fixed

**Comment by mpodwysocki** [comment]
> Did we already make the API version change related to adding `userId` to the Installation body?

Yes, that `api-version` was changed to `2020-06` for version 4.0 to support throttling.

**Pull request merged**